ChangeLog.md

```diff
@@ -7,6 +7,10 @@
    the width (vertical/horizontal or Euclidean) of a convex hull.
    (Bertrand Kerautret, [#1052](https://github.com/DGtal-team/DGtal/pull/1052))
 
+ - MelkmanConvexHull: addition of a reverse method in order to process points
+   coming from front or back in reference to a digital contour.
```

Comment:
"New method to allow point insertions and convex hull computation on both side of a point sequence " ?